canisters/src/vts/Cargo.toml

```diff
@@ -11,6 +11,9 @@ candid = { version = "0.10.7", features = [], default-features = false }
 ic-cdk = { version = "0.13.2", features = [], default-features = false }
 ic-stable-structures = { version = "0.6.4", features = [], default-features = false }
 serde = { version = "1.0.198", features = [], default-features = false }
+ic-cdk-macros = { version = "0.13.2", features = [], default-features = false }
+rust_decimal = { version = "1.35.0", features = ["serde-with-str"] }
```

Comment:
Can you put please one more option here: default-features = false? It seems you don't need default features?